.github/workflows/bun-linux-aarch64.yml

```diff
@@ -48,7 +47,7 @@ jobs:
             tag: linux-aarch64
             arch: aarch64
             build_arch: arm64
-            runner: linux-arm64
+            runner: ubicloud-standard-16-arm
```

Comment:
reading the concurrency limits on https://www.ubicloud.com/docs/github-actions-integration/price-performance#concurrency-limits sounds like you should not have too much `-16` jobs in parallel, as they seem to share a total of 64 CPUs

Reply:
Hi there, I'm Ozgun, one of Ubicloud's founders. If you drop us an email at support@ubicloud.com about concurrency limits, we'd be happy to help you out.